compact/far_forward/roman_pots_eRD24_design.xml

```diff
@@ -76,7 +76,7 @@
             <position x="ForwardRomanPotStation1_xpos" y="0" z="ForwardRomanPotStation1_zpos" />
             <rotation x="0" y="ForwardRomanPotStation1_rotation" z="0" />
             <module name="ModuleRP1" vis="FFTrackerShieldedModuleVis" nx="1" ny="1" width="ForwardRomanPot_ModuleWidth" height="ForwardRomanPot_ModuleHeight">
-                <module_component material="Aluminum"     vis="FFTrackerShieldingVis" thickness="ForwardRomanPot_RFShieldThickness"/>
+                <!--<module_component material="Aluminum"     vis="FFTrackerShieldingVis" thickness="ForwardRomanPot_RFShieldThickness"/>-->
```

Comment:
Please remove or provide comment on when and why this will have to be reintroduced. Definition of ForwardRomanPot_RFShieldThickness may need to be removed.